pages/releases/4.3.html

```diff
@@ -0,0 +1,2109 @@
+										</code>
+										property hint
+									</a>
+								</h4>
+								<div class="c-content">
+									<p>
+										You need specific types of export? We got you covered.
+									</p>
+									<p>
+										You can now use
+										<code class="highlight">
+											<span class="gdscript-annotation">@export_custom</span>
+										</code>
+										to export a value suited to your needs.
+									</p>
+									<p>
+										It allows you to set a custom hint, hint string, and usage flags for the
+										exported property. Great for highly specific exports and for addon developers.
+									</p>
+								</div>
+								<div class="c-link"
+									data-contributors="vnen"
+									data-read-more="https://github.com/godotengine/godot/pull/72912">
+								</div>
+							</div>
+						</div>
+					</div>
+				</div> <!-- #gdscript-export-custom-property-hint -->
+
+				<div id="gdscript-autocompletion-fixes" class="release-card span-2-at-3-col">
+					<div class="release-card-container">
+						<div class="release-card-content">
+							<div class="release-card-content-container">
+								<h4 class="c-title">
+									<a href="#gdscript-autocompletion-fixes">
+										GDScript autocompletion fixes
+									</a>
+								</h4>
+								<div class="c-blockquote">
+									Do you need a <span class="highlight">suggestion</span>?
+								</div>
+								<div class="c-content">
+									<p>
+										We improved the autocompletion of several components, including enums,
+										subscripts, and
+										<code
+											class="highlight"><span class="function">get_node</span><span class="symbol">()</span></code>.
+									</p>
+									<p>
+										As it is a difficult feature to detect regressions, we even started doing some
+										automated testing to specifically test autocompletion results.
+									</p>
+								</div>
+								<div class="c-link"
+									data-contributors="HolonProduction"
+									data-read-more="https://github.com/godotengine/godot/pull/86973">
+								</div>
+							</div>
+						</div>
+					</div>
+				</div> <!-- #gdscript-autocompletion-fixes -->
+
+				<div id="gdscript-circular-dependencies-between-scenes-and-scripts" class="release-card">
+					<div class="release-card-container">
+						<div class="release-card-content">
+							<div class="release-card-content-container">
+								<h4 class="c-title">
+									<a href="#gdscript-circular-dependencies-between-scenes-and-scripts">
+										GDScript circular dependencies between scenes and scripts
+									</a>
+								</h4>
+								<div class="c-content">
+									<p>
+										A lot of issues with circular dependencies between scenes and scripts have
+										been fixed!
+									</p>
+									<p>
+										Not only this makes you less worry about cyclic dependencies, but it makes
+										scenes more resilient and stable.
+									</p>
+								</div>
```

Comment:
```suggestion
								<h4 class="c-title">
									<a href="#gdscript-circular-dependencies-between-scenes-and-scripts">
									 On circular dependencies
									</a>
								</h4>
								<div class="c-content">
									<p>
										Circular dependencies (also known as cyclic dependencies) happen when two scripts call upon one another. While this might be considered an anti-pattern in software engineering, in reality it is a rather common occurence.
									</p>
									<p>
										We fixed some known issues in GDScript that would show up in connection with such circular dependencies, to allow you to worry less about them in your projects. These changes will make your scenes more resilient and stable in general.
									</p>
								</div>
```